### PR TITLE
Integration tests for Applications-Nav - PLAT-4404

### PR DESCRIPTION
**Pull request by lmacedodaitan**

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (vpopolin) [inline .idea/vcs.xml]: This file is IDE-specific, whe shouldn't version it.

File: apps/extensions-api-test/extensions-api-test.js [hunk 2]
```diff
@@ -36,7 +36,7 @@ exports.getApplication = function(req, res)
 	app.addVariable('appId', CONFIG['extensions-api-test'].appId);
 	app.addVariable('controller', CONFIG['extensions-api-test'].controller);
 	app.addVariable('moduleApp', CONFIG['extensions-api-test-module'].appId);
-	app.addVariable('authenticate', CONFIG['extensions-api-test'].auth !== 'undefined');
```
Comment (vpopolin) [inline apps/extensions-api-test/extensions-api-test.js]: This was a workaround to skip the auth flow, but cannot be commited this way.

File: config/config.dev.js [hunk 3]
```diff
@@ -14,16 +14,11 @@ module.exports = {
 		key : fs.readFileSync(__dirname + '/certs/localhost.key', {encoding: 'utf-8'}),
 		cert : fs.readFileSync(__dirname + '/certs/localhost.cert', {encoding: 'utf-8'}),
 	},
-	auth: {
```
Comment (vpopolin) [inline config/config.dev.js]: This was a workaround to skip the auth flow, but cannot be commited this way. Keep the change from nexus1-1 to nexus2-2.

File: spec/navSpec.js [hunk 4]
```diff
@@ -75,7 +75,6 @@ describe('NAVIGATION', function() {
     it('should select when clicked', function(done) {
         function select() {
             return this.driver.executeAsyncScript(function() {
-                debugger;
```
Comment (vpopolin) [inline spec/navSpec.js]: Remove this line.

File: .idea/vcs.xml [hunk 5]
```diff
@@ -0,0 +1,6 @@
...
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="" vcs="Git" />
+  </component>
+</project>
```
Comment (Ondoher) [inline .idea/vcs.xml]: What is this file? should it be added to .gitignore

File: spec/test.js [hunk 6]
```diff
@@ -3,7 +3,7 @@ var selenium = require('selenium-webdriver');
 var logging = selenium.logging;
 
 jasmine.getEnv().addReporter(new Reporter());
-jasmine.DEFAULT_TIMEOUT_INTERVAL = 20000;
+jasmine.DEFAULT_TIMEOUT_INTERVAL = 60000;
```
Comment (vpopolin) [inline spec/test.js]: Is this increase in timeout really needed?

File: spec/test.js [hunk 7]
```diff
@@ -16,7 +16,7 @@ afterEach(function(done) {
         .then(function(logs) {
             if (config.log) {
                 logs.forEach(function(log) {
-                    console.log(log.message);
+                    //console.log(log.message);
```
Comment (vpopolin) [inline spec/test.js]: Uncomment.

File: spec/test.js [hunk 8]
```diff
@@ -28,4 +28,3 @@ require('./loginSpec');
 require('./loaderSpec');
 
 require('./navSpec');
-require('./uiSpec');
```
Comment (Ondoher) [inline spec/test.js]: Why did we remove this?

Comment (vpopolin) [reply spec/test.js]: Those tests were broken, Lucas is working on them in his current task.

**Pull request merged**